Add non third party dependancy management

tombye opened the pull request:
In response to this issue https://github.com/TheDoubleK/digitalmarketplace-thermos/issues/10, this adds a script to handle the management of our non-third-party front-end dependencies.

The script is run by NPM after it install its dependencies (when you run `npm install`).

TheDoubleK commented:
I have tried running `npm install` and `node_modules/gulp/bin/gulp.js build:development` but trying to access the /test page I get the error "node_modules/gulp/bin/gulp.js build:development".

Is there something else I need to do to get it to work?

Also, the README should be updated now that the `scripts/update-dm-toolkit.sh` script no longer exists.

tombye commented:
Sorry, I changed the name of the govuk template folder from `jinja_govuk_template` to `govuk_template` without updating all the references to this. This is what caused the error you saw.

I've also rewritten the front-end part of `./README.md` to better describe all the dependency management stuff.

TheDoubleK commented:
Nice one - looking good again!
